_spinnaker_user_guides/plugin-creators.md

```diff
@@ -193,25 +207,29 @@ window.spinnakerSettings.onPluginLoaded(plugin);
 ```
 
 **Render Method**
-Anything can go in the render method. What is in the render method will be shown to plugin users when configuring their Spinnaker pipeline. In this example, the user can input the maximum number of seconds to wait to continue executing the pipeline.
+
+Anything can go in the render method. What is in the render method is shown to plugin users when configuring their Spinnaker pipeline. In this example, the user can input the maximum number of seconds to wait before continuing to execute the pipeline.
 
 **Set Methods**
-Stages are made up of JSON that contains all information that will be passed to the backend. To update the stage JSON with the data the user enters use `this.props.updateStageField` method that takes in a valid JSON object of what to update. In this example we are updating the `maxWaitTime` field with the value that the user enters.
+
+Stages are made up of JSON that contains all the information that gets passed to the backend. To update the stage JSON with the data the user enters, use `this.props.updateStageField` method, which takes in a valid JSON object of what to update. In this example, we update the `maxWaitTime` field with the value that the user enters.
 
 **Register Stage**
-The `registerStage` method is what makes the stage available to be used. These are the required fields for registering a stage.
 
-1. key → a unique name of the stage
-2. label → is what is used inside the UI to display, saying what the name of the stage is
-3. description → a short description of what the stage will do
-4. component → if using React to create a stage, this is where you would put the component to render
+The `registerStage` method is what makes the stage available for use. These are the required fields for registering a stage:
+
+* key → a unique name of the stage
+* label → is what is used inside the UI to display, saying what the name of the stage is
+* description → a short description of what the stage will do
+* component → if using React to create a stage, this is where you would put the component to render
 
 Optional Fields:
+* cloudProvider → if the stage can only be ran in one of the cloud providers, that can be selected here
```

Comment:
How come cloudProvider is an optional field for a plugin? is it part of how a stage is registered?

Reply:
there doesn't have to be a cloudprovider for a stage. a stage could work on any cloud, like manual judgement